openshift/template.yaml

```diff
@@ -43,45 +63,6 @@ objects:
                     secretKeyRef:
                       name: coreapi-postgres
                       key: host
-                - name: POSTGRESQL_PORT
-                  valueFrom:
-                    secretKeyRef:
-                      name: coreapi-postgres
-                      key: port
-                - name: POSTGRESQL_DATABASE
-                  valueFrom:
-                    secretKeyRef:
-                      name: coreapi-postgres
-                      key: database
-                - name: POSTGRESQL_PASSWORD
-                  valueFrom:
-                    secretKeyRef:
-                      name: coreapi-postgres
-                      key: password
-                - name: POSTGRESQL_USER
-                  valueFrom:
-                    secretKeyRef:
-                      name: coreapi-postgres
-                      key: username
-                - name: AWS_SQS_ACCESS_KEY_ID
-                  valueFrom:
-                    secretKeyRef:
-                      name: aws
-                      key: sqs-access-key-id
-                - name: AWS_SQS_SECRET_ACCESS_KEY
-                  valueFrom:
-                    secretKeyRef:
-                      name: aws
-                      key: sqs-secret-access-key
-                - name: PGBOUNCER_SERVICE_HOST
-                  value: bayesian-pgbouncer
-              resources:
```

Comment:
Is this resource requirements removed by accident?

Reply:
Not doing anything with pgbouncer now in the gh refresh job. hence removed.

Reply:
You removed resources section beneath this line which needs Memory and CPU request and limit numbers

Reply:
bad miss. will add it.